### PR TITLE
Writing efields

### PR DESCRIPTION
**Pull request by anelles**
Remove the efield stuff from channel. It is not clear to me, when whether there is actually a difference between mini and micro now ...

### CODE REVIEW

**Comment by anelles** [comment]
We need to discuss what to do about file size and make mini, micro and full better distinct. 250 KB per event without efields is huge for an array of 100 stations.

File: NuRadioReco/framework/channel.py [hunk 1]
```diff
@@ -55,9 +55,6 @@ def serialize(self, mode):
         data = {'parameters': NuRadioReco.framework.parameter_serialization.serialize(self._parameters),
                 'id': self.get_id(),
                 'base_trace': base_trace_pkl}
-        if(self.__electric_field is not None):
-            if(not(mode == 'micro' or mode == 'mini')):
-                data['electric_field'] = self.__electric_field.serialize()
```
Comment (cg-laser) [inline NuRadioReco/framework/channel.py]: we should remove the `self.__electric_field` from everywhere

**Comment by anelles** [comment]
Right now, it looks like 'full' stores everything, 'mini' only the efield and 'micro' only the channel. I find this backwards, I think the efields are least important. Any thoughts on this?

**Comment by cg-laser** [comment]
for our cases we need more modes. Once we have a neutrino efield reco, the reconstructed efields are more important than the voltage traces. But we often don't need the simulated efields... Can we come up with a better scheme to select individual components? Maybe just arguments such as `save_sim_efield=True`, `save_voltage_traces=True`, `save_efields=True`

But we should do this in a fresh pull request, otherwise we might get merge conflicts with other changes to the framwork classes

**Pull request merged**